coast/internal_tide.py

```diff
@@ -113,23 +112,24 @@ def construct_pycnocline_vars(self, nemo_t: xr.Dataset, nemo_w: xr.Dataset, stra
 
         """
         debug(f"Constructing pycnocline variables for {get_slug(self)}")
-        #%% Contruct in-situ density if not already done
+        # Construct in-situ density if not already done
         if not hasattr(nemo_t.dataset, "density"):
             nemo_t.construct_density(EOS="EOS10")
 
-        #%% Construct stratification if not already done. t-pts --> w-pts
+        # Construct stratification if not already done. t-pts --> w-pts
         if not hasattr(nemo_w.dataset, "rho_dz"):
             nemo_w = nemo_t.differentiate(
                 "density", dim="z_dim", out_varstr="rho_dz", out_obj=nemo_w
-            )  # --> sci_nwes_w.rho_dz
+            )  # TODO These kwargs don't appear to exist
```

Comment:
Probably my bad notation. Just delete this comment line 124

Reply:
@jpolton I think this comment is about the xarray.Dataset.differentiate() method not having a 'dim', 'out_varstr' or 'out_obj' arguments:
![image](https://user-images.githubusercontent.com/64262641/134169416-156e71b9-6f6e-49e3-973b-b00514e41f32.png)

So I'm assuming this method call would error out?